packages/cli/src/sso/saml/errors/invalid-saml-metadata.error.ts

```diff
@@ -0,0 +1,7 @@
+import { ApplicationError } from 'n8n-workflow';
+
+export class InvalidSamlMetadataError extends ApplicationError {
+	constructor() {
+		super('Invalid SAML metadata', { level: 'error' });
```

Comment:
This wll be reported to Sentry. Is this actionable on our end? Else let's make it `warning`

Reply:
Yeah, thought that maybe some of them set up their on sentry to track this. I'll change it to warning.